OntologyMapper/test/OntologyMappingManagerTests.cs

```diff
@@ -0,0 +1,215 @@
+
+            Assert.True(result);
+            Assert.NotNull(propertyProjection);
+            Assert.Equal(outputDtmiFilter, propertyProjection.OutputDtmiFilter);
+            Assert.Equal(outputPropertyName, propertyProjection.OutputPropertyName);
+            Assert.Equal(inputPropertyName, propertyProjection.InputPropertyName);
+        }
+
+        [Fact]
+        public void TryGetPropertyProjection_ReturnsFalse_When_NotFound()
+        {
+            var mockOntologyLoader = new Mock<IOntologyMappingLoader>();
+            mockOntologyLoader.Setup(m => m.LoadOntologyMapping()).Returns(GetOntologyMapping);
+
+            var ontologyMappingManager = new OntologyMappingManager(mockOntologyLoader.Object);
+
+            var outputDtmiFilter = "*";
+            var outputPropertyName = "external";
+
+            var result = ontologyMappingManager.TryGetPropertyProjection(outputDtmiFilter, outputPropertyName, out var propertyProjection);
+
+            Assert.False(result);
+            Assert.Null(propertyProjection);
+        }
+
+        [Fact]
+        public void TryGetFillProperty_ReturnsTrue_When_Found()
+        {
+            var mockOntologyLoader = new Mock<IOntologyMappingLoader>();
+            mockOntologyLoader.Setup(m => m.LoadOntologyMapping()).Returns(GetOntologyMapping);
+
+            var ontologyMappingManager = new OntologyMappingManager(mockOntologyLoader.Object);
+
+            var outputDtmiFilter = "*";
+            var outputPropertyName = "name";
+
+            var result = ontologyMappingManager.TryGetFillProperty(outputDtmiFilter, outputPropertyName, out var propertyFill);
+
+            Assert.True(result);
+            Assert.NotNull(propertyFill);
+            Assert.Equal(2, propertyFill.Count());
+            Assert.Equal("name", propertyFill.ToList()[0]);
+            Assert.Equal("description", propertyFill.ToList()[1]);
+        }
+
+        [Fact]
+        public void TryGetFillProperty_ReturnsFalse_When_NotFound()
+        {
+            var mockOntologyLoader = new Mock<IOntologyMappingLoader>();
+            mockOntologyLoader.Setup(m => m.LoadOntologyMapping()).Returns(GetOntologyMapping);
+
+            var ontologyMappingManager = new OntologyMappingManager(mockOntologyLoader.Object);
+
+            var outputDtmiFilter = "*";
+            var outputPropertyName = "channel";
+
+            var result = ontologyMappingManager.TryGetFillProperty(outputDtmiFilter, outputPropertyName, out var propertyFill);
+
+            Assert.False(result);
+            Assert.NotNull(propertyFill);
+            Assert.Empty(propertyFill);
+        }
+
+        private OntologyMapping GetOntologyMapping()
+        {
+            var ontologyMapping = new OntologyMapping();
+
+            ontologyMapping.Header.InputOntologies.Add(new Ontology { DtdlVersion = "v0", Name = "mapped", Version = "1.0" });
```

Comment:
For clarity since this is test code, it might be clearer to make these not real values